### ISSUE TITLE
[BUG] Undefined HIFI_API_VERSION in npm version

### ISSUE THREAD

**Issue by jjv360** [highfidelity/hifi-spatial-audio-js#43]
## Describe the Bug
Hi, as soon as I import the library from npm, it throws an error about HIFI_API_VERSION undefined. In order to workaround the issue, I added this to the top of the <head> in the html:

```html
<script>
    window.HIFI_API_VERSION = window.HIFI_API_VERSION || ''
</script>
```

I'm not sure if this is specific to our project, but it doesn't seem so...

## Steps to Reproduce
1. From a React project built with Webpack, do: `import { HiFiCommunicator } from 'hifi-spatial-audio'`
3. See error.

## Expected Behavior
Class imported correctly.

## User or Developer Environment
- Environment: Web
- NodeJS Version: v14.15.5
- OS: Mac
- Browser: Chrome
- Library version: 0.6.0 and 0.6.1-1

## Screenshots
<img width="580" alt="Screenshot 2021-03-23 at 16 59 10" src="https://user-images.githubusercontent.com/2771104/112167466-26436280-8bf9-11eb-83a5-d48a23636695.png">

## Additional Information
None

**Comment by zfox23** [comment]
Thanks for the bug report, @jjv360! I'll take a look at this today.

**Comment by zfox23** [comment]
@jjv360 - a couple things:
1. Thanks again!
2. This should be fixed with `hifi-spatial-audio` v0.6.1-6 and later. Please update your `hifi-spatial-audio` package locally; v0.6.1-6 is currently installable via `npm i hifi-spatial-audio@main`.
3. Inside our examples repository, I've added a "Webpack" example to help folks get started using the Spatial Audio API with Webpack (or React, or etc...). [Check out that example here.](https://github.com/highfidelity/Spatial-Audio-API-Examples/tree/main/examples/nodejs/webpack) I thought that might come in handy for you.

I'm going to close this issue now - please re-open it if you're still encountering this bug.

**Comment by jjv360** [comment]
Thanks, it is working now 👍
